docs/features/reuse.md

```diff
@@ -0,0 +1,34 @@
+# Reusable Containers (Experimental)
+
+!!! warning 
+    Reusable containers is still an experimental feature and the behavior can change.
+    Those containers won't stop after all tests are finished.
+
+Reusable containers feature keeps the containers running between the test sessions. In order
+to achieve it, manual initialization should be used by calling `start()`
+and should be enabled by environment. In order to reuse a container, the
+configuration *must not change*.
+
+!!! note
+    Reusable containers doesn't suit for CI usage and as an experimental feature
+    not all Testcontainers features are working such as resources cleanup
+    or networking.
+
+## How to use it
+
+* Define container with `withReuse(true)`
+
+```java
+GenericContainer container = new GenericContainer("redis:6-alpine")
```

Comment:
Ideally we have a code include, but I think for such a small snippet, it is fine 😉